### PR TITLE
Update community supported README: legal disclaimer

### PR DESCRIPTION
**Pull request by cwarnermm**
Added legal disclaimer to README, as requested by Product.

### CODE REVIEW

**Review by hanzei — CHANGES_REQUESTED** [review]
Blocking until the discussion in https://community.mattermost.com/private-core/pl/51hbdxfn7tbbuboys4siw8peea has been resolved

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# :warning: Disclaimer
+
+**This repository is community supported and not maintained by Mattermost. Mattermost disclaims any and all liability for integrations, including Third Party Integrations and Mattermost Integrations. All integrations are provided "AS IS", and may be used at your own risk. Integrations may be modified or discontinued at any time.**
```
Comment (cwarnermm) [inline README.md:3]:
```suggestion
# Disclaimer

**This repository is community supported and not maintained by Mattermost. Mattermost disclaims liability for integrations, including Third Party Integrations and Mattermost Integrations. Integrations may be modified or discontinued at any time.**
```

**Comment by cwarnermm** [comment]
@hanzei - Legal's updated guidance and language for the legal disclaimer for community-supported integrations has been applied to this PR.

**Comment by cwarnermm** [comment]
@hanzei - This PR is ready for your final approval.

**Review by hanzei — APPROVED** [review]

**Pull request merged**